Functions.Templates/Templates/SqlInputBinding-CSharp-Isolated/.template.config/template.json

```diff
@@ -0,0 +1,71 @@
+{
+    "author": "Microsoft",
+    "classifications": [
+        "Azure Function",
+        "Input",
+        "SQL"
+    ],
+    "name": "SqlInputBindingIsolated",
+    "identity": "Azure.Function.CSharp.Isolated.SqlInputBinding",
+    "groupIdentity": "Azure.Function.SqlInputBindingIsolated",
+    "shortName": "sqlinput",
```

Comment:
Should this be `sqlinputisolated`?

Reply:
I've referenced cosmos, signalr and servicebus isolated templates, all of them have the same attribute name without isolated being tagged. I guess adding we  could do our own thing here and do sqlinputisolated, @chlafreniere, @dzsquared any recommendations?

Reply:
I don't have strong feelings either way - the template.json schema and field purposes is greek to me.

Reply:
If other extensions don't put `isolated` in the shortName, then let's keep it as is IMO.